### DISCUSSION TITLE
[Feature] Shortcut for the zotero-pdf-preview plugin

### DISCUSSION THREAD

**Discussion by mjthoraval** [windingwind/zotero-actions-tags#171] [Ideas]
I would like to use a shortcut to toggle the Preview pane in the Info Tab for the [zotero-pdf-preview](https://github.com/windingwind/zotero-pdf-preview) plugin.
I am not sure if this should wait for the update of that plugin, or if there is a simple way to implement it here?

**Comment by windingwind** [comment]
See #172

**Comment by mjthoraval** [comment]
Thank you.
I understand that the action you have provided allows switching between tabs, which is very nice.
However, my question was more targetted to the preview in the split view of the Info Tab.
Is that possible to implement also?

**Reply by windingwind** [reply]
Well, the current layout of the preview split is quite buggy (and that's why I didn't make a shortcut in Z7 version), which makes it not very possible to collapse the split (I tried, but it's also buggy).

The layout of the item pane is going to have a significant change. So before Z7 is out, I don't think we will spend more time on that layout.

**Reply by mjthoraval** [reply]
Thank you. I will wait for the release of Z7.
